app/assets/javascripts/templates/enterprise_modal.html.haml

```diff
@@ -1,5 +1,2 @@
-%ng-include{src: "'partials/enterprise_header.html'"}
-%ng-include{src: "'partials/enterprise_details.html'"}
-%ng-include{src: "'partials/hub_details.html'"}
```

Comment:
I think I would rename these two templates `enterprise_modal` for the regular one and `enterprise_list_modal` for the modal with multiple enterprises.

Reply:
Ah yes, that does sound better :)